### DISCUSSION TITLE
redirect

### DISCUSSION THREAD

**Discussion by wakiullah** [community/community#136227] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

When i visit my repository live link https://wakiullah.github.io/ecom/ it's redirect to http://wakiullah.me/ecom/ and showing this message "This site can’t be reached".. now how to setup my setting to stop this redirect.

**Comment by tharinm** [comment]
If you added custom domains remove it and check.
Make sure the "Enforce HTTPS" option is checked. This ensures that the site serves securely over https://.
